techniques/flash/FLASH14.html

```diff
@@ -11,7 +11,7 @@ <h3>Updating button text with multiple event handlers</h3>
          
             <p>In this example, a group of buttons is assigned the same event   handlers for the flash.events.FocusEvent.FOCUS_IN and flash.events.MouseEvent.MOUSE_OVER   events. When a button receives focus or is hovered over using a mouse,   text describing the button will be updated. </p>
          
-         <pre xml:space="preserve">import fl.accessibility.ButtonAccImpl;
+         <pre>import fl.accessibility.ButtonAccImpl;
```

Comment:
```suggestion
         <pre><code>import fl.accessibility.ButtonAccImpl;
```

and matching closing in https://github.com/w3c/wcag/pull/4611/files#diff-d95a61bcc2ec3cfb7330e5089d7558ffc3c219a6c52c1e6d2010d715ae0efd14R65